src/github.com/couchbaselabs/sync_gateway/base/bucket.go

```diff
@@ -54,6 +58,10 @@ func (bucket couchbaseBucket) GetName() string {
 	return bucket.Name
 }
 
+/*func (bucket couchbaseBucket) GetUUID() string {
```

Comment:
Can we just delete this commented code if its not serving any purpose?  Or if it will definitely serve a purpose in the future, just uncomment it?